src/indexer/index.ts

```diff
@@ -15,16 +15,17 @@ import {
 } from "@algo-builder/algob/build/types";
 import type { LogicSigAccount } from "algosdk";
 import { getProposalLsig } from "../contract/dao";
+import indexerClient from "../config/indexer.config";
```

Comment:
Same like `config/indexer.config` only work with local network. 
I recommend we import those value from env.
Example: 
```javascript
const indexer_token = process.env.INDEXER_TOKEN || "";
const indexer_server = process.env.INDEXER_URL || "http://localhost";
const indexer_port = process.env.INDEXER_PORT || 8980;
```

Reply:
Added